packages/application-shell/package.json

```diff
@@ -118,7 +118,8 @@
     "react-dom": ">=16 <17",
     "react-intl": ">=2 <3",
     "react-router": ">=4 <5 || >=4.4.0-beta.0 <5",
-    "react-router-dom": ">=4 <5 || >=4.4.0-beta.0 <5"
+    "react-router-dom": ">=4 <5 || >=4.4.0-beta.0 <5",
+    "react-testing-library": ">=5 || <6"
```

Comment:
I moved this as a peer dep because in `jest-preset-mc-app` it was already defined as such.

Reply:
Should solve us not being able to update in the mc-fe.